metrics/anls/requirements.txt

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/huggingface/evaluate
```

Comment:
Can you add the same placeholder as in the other metrics?

Reply:
done